### DISCUSSION TITLE
LFI tags doesn't fit the URL and parameter.

### DISCUSSION THREAD

**Discussion by Lukmanern** [projectdiscovery/nuclei#3930] [Q&A]
So I performed a scan using the following format:
nuclei -u "http://testphp.vulnweb.com/showimage.php?file=FUZZ-HERE" -tags lfi
However, Nuclei couldn't provide any fuzzing results, and it may not have even attempted it on the URL parameter.

When I tried a different link, like performing Fuzzing correctly with Nuclei:

nuclei -u "http://testphp.vulnweb.com/showimage.php?file=../../etc/passwd" -tags lfi

I think there's an answer to this.

Thank you
Regards, Lukman.

<img width="471" alt="zyc" src="https://github.com/projectdiscovery/nuclei/assets/48827979/3298c044-c680-4bbf-b6ae-97db450bb167">

**Comment by ehsandeep** [comment]
@Lukmanern you need to run fuzzing templates from here - https://github.com/projectdiscovery/fuzzing-templates, default project contains templates for known cves / vulnerabilities.

**Reply by Lukmanern** [reply]
I just wanna scan for LFI with fuzzing/linux-lfi-fuzzing.yaml, but why so many templates load? Is there any proper example to 'exclude/include' tags? (6312 templates)

i've seen this before https://github.com/projectdiscovery/nuclei/discussions/1337


![Screenshot (2631)](https://github.com/projectdiscovery/nuclei/assets/48827979/f2109f62-f1b1-4444-b6fc-adedf2093dd8)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
